### ISSUE TITLE
Change bank display system to match current time

### ISSUE THREAD

**Issue by PearlyPrimm** [kylemcshea/homestead-rsg#10]
![Image](https://github.com/kylemcshea/homestead-rsg/assets/132966196/ec53e5b4-7e17-4e72-8de8-893708edb40a)

**Comment by kalekaneki** [comment]
Yeah this needs to definitely be changed to something more from 1800s haha.
